services/src/main/java/org/keycloak/services/resources/LoginActionsService.java

```diff
@@ -1044,7 +1046,10 @@ public void ignore() {
         } else if (context.getStatus() == RequiredActionContext.Status.FAILURE) {
             response = interruptionResponse(context, authSession, action, Error.CONSENT_DENIED);
         } else {
-            throw new RuntimeException("Unreachable");
+            if (!Objects.equals(factory.getId(), DeleteAccount.PROVIDER_ID)) {
+                throw new RuntimeException("Unreachable");
+            }
+            response = Response.status(302).location(Urls.accountPage(session.getContext().getUri().getBaseUri(), realm.getName())).build();
```

Comment:
There shouldn't be a need for any changes in here. Can you elaborate on why you are doing this?

Reply:
Good question. I also thought twice before adding this. The reason is if I use `context.success()` it will enter the first case above.  And there are a number of potential NPE that can happen (already tried) because the user is already deleted by then. The first one is  `authSession.getAuthenticatedUser().removeRequiredAction(factory.getId());`